modules/core/nuxeo-core-el/src/main/java/org/nuxeo/ecm/platform/el/ExpressionContext.java

```diff
@@ -51,7 +62,13 @@ public ValueExpression setVariable(String variable, ValueExpression expression)
 
         @Override
         public Method resolveFunction(String prefix, String localName) {
-            return map.get(prefix + ":" + localName);
+            String key = String.format("%s:%s", prefix, localName);
```

Comment:
`String.format` is overkill and costly (and you don't use it in `mapFunction` anyway).